### DISCUSSION TITLE
Is NS Vue actively maintained?

### DISCUSSION THREAD

**Discussion by bgrand-ch** [nativescript-vue/nativescript-vue#907] [Q&A] [answered]
- Vue.js 3?
- Official Vue Router?
- Updated docs on nativescript-vue.org?
- Issues tracked and resolved?

Thanks for all your works.

**Comment by rigor789** [comment, marked as answer]
- Vue 3 - coming, slower than I'd like, but we'll get there!
- Router? Maybe in v3, will need some experimentation again - in 2.x it ended up being unfeasible
- Most of the docs should be up-to-date on the ns site - the ones we have on ns-vue site should be up-to-date, though probably missing a few things.
- There hasn't been too many issues popping up, and those that have - have been fixed mostly. The rest are generally questions. If there are any speicifc issues you are referring to, let me know - I could've easily missed some...

There haven't been many commits on the repo, but that doesn't meant it's inactive - it's been pretty stable, didn't require too much fixing...

**Reply by bgrand-ch** [reply]
Thanks a lot for your detailled answer and for all your hard works.

I will use NS Vue for my next project 😉

**Reply by fanckush** [reply]
almost 3 years later, did anything change in the plans?

**Reply by rigor789** [reply]
Vue 3 is in RC, used in production apps, working on a new landing page/docs for the final release. Anything specific you're interested in @fanckush?

**Reply by fanckush** [reply]
Not in particular, thanks for the reply and reassurance.
Having never used nativescript before, I can’t say for sure until I try it.
Eagerly waiting for the docs to be ready 👀
